modules/aws/lambda.go

```diff
@@ -59,6 +59,30 @@ func (err *FunctionError) Error() string {
 	return fmt.Sprintf("%s error invoking lambda function: %v", err.Message, err.Payload)
 }
 
+//GetLambdaFunctionConfiguration gets ARN for a given function.
+func GetLambdaFunctionConfiguration(t testing.TestingT, region, functionName string) lambda.FunctionConfiguration {
+	out, err := GetLambdaFunctionArnE(t, region, functionName)
+	require.NoError(t, err)
+	return out
+}
+
+//GetLambdaFunctionConfigurationE gets ARN for a given function.
+func GetLambdaFunctionConfigurationE(t testing.TestingT, region, functionName string) (lambda.FunctionConfiguration, error) {
+	lambdaClient, err := NewLambdaClient(t, region)
+
+	input := &lambda.GetFunctionInput{
+		FunctionName: &functionName
+	}
+
+	out, err = lambdaClient.GetFunction(input)
+	require.NoError(t, err)
```

Comment:
Remove this line, as the `E` flavor should be bubbling errors instead of failing tests.